sknn/mlp.py

```diff
@@ -309,6 +309,21 @@ def _create_mlp(self, X, y, nvis=None, input_space=None):
             seed=self.random_state,
             input_space=input_space)
 
+    def _create_matrix_input(self, X, y):
+        if self.is_convolution:
+
+            # b01c arrangement of data
+            # http://benanne.github.io/2014/04/03/faster-convolutions-in-theano.html for more
+            #input: (batch size, channels, rows, columns)
```

Comment:
Add  space after the # in the comment.  And you can remove multiple empty lines, one empty line here and there is down to style :-)